tests/MigrationGeneratorTest.php

```diff
@@ -72,9 +74,40 @@ public function testCreateMigrationMYSQL()
                 'string' => ['title', 'body'],
                 'json' => ['meta'],
                 'timestamp' => ['created_at'],
+                'timestamp-required' => ['published_at'],
             ])
             ->generate();
 
         $this->assertGeneratedFileEquals('migrations_mysql.php', 'database/migrations/2022_02_02_000000_posts_create_table.php');
```

Comment:
let's rename fixture to

```suggestion
        $this->assertGeneratedFileEquals('generated_mysql_migration.php', 'database/migrations/2022_02_02_000000_posts_create_table.php');
```